react-native-reanimated-tests.tsx

```diff
@@ -499,3 +501,20 @@ function interpolateNodeTest() {
     outputRange: ['0deg', '100deg'],
   });
 }
+
+function colorTest() {
+  const r = useValue(255);
+  const g = useValue(255);
+  const b = useValue(255);
+  const a = useValue(255);
+  return color(r, g, b, a);
+}
+
+function interpolateColorsTest() {
+  const animationValue = useValue(0);
+  const color = interpolateColors(animationValue, {
+    inputRange: [0, 1],
+    outputColorRange: ['red', 'blue'],
```

Comment:
This won't work in the real code, we only return string RGB values in the format of `rgba(r,g,b,a)` (where variables are numbers) on the web.

Reply:
then, what it would be like?

Reply:
Or actually, it'll work, we use `processColor` internally. I mixed `outputColorRange` with the return type, my bad.